taglib/fileref.cpp

```diff
@@ -54,6 +54,62 @@
 
 using namespace TagLib;
 
+template <typename T>
+static File* createInternal(T arg, const String& ext, bool readAudioProperties,
```

Comment:
I'm fine with passing the filename twice, of course. That's why I suggested it. :)